ansible_base/lib/dynamic_config/dynamic_settings.py

```diff
@@ -139,6 +140,8 @@
     SOCIAL_AUTH_STRATEGY = "ansible_base.authentication.social_auth.AuthenticatorStrategy"
     SOCIAL_AUTH_LOGIN_REDIRECT_URL = "/"
 
+    # URL to send users when social auth login fails
+    LOGIN_ERROR_URL = "/"
```

Comment:
Value for this TBD depending what UI does.  Also, maybe this should go in aap-gateway instead?

Reply:
Agreed upon /?auth_failed